.azure/pipelines/helix-matrix.yml

```diff
@@ -48,3 +52,41 @@ jobs:
     - name: Helix_logs
       path: artifacts/log/
       publishOnError: true
+      
+# Helix ARM64
+- ${{ if ne(variables['System.TeamProject'], 'internal') }}:
+  - template: jobs/default-build.yml
+    parameters:
+      jobName: Helix_matrix_arm64
+      jobDisplayName: "Tests: Helix ARM64 matrix"
+      agentOs: Windows
+      timeoutInMinutes: 480
+      steps:
+      - script: ./eng/build.cmd
+                -ci
+                -arch x64
+                -pack
+                -all
+        displayName: Build x64/arm64      
+      # Build the arm64 shared framework
+      - script: ./eng/build.cmd
+                -ci
+                -arch arm64
+                -pack
+                -noBuildJava
+                -noBuildNative
+                /p:OnlyPackPlatformSpecificPackages=true
+        displayName: Build ARM64 shared fx
+      - script: .\eng\build.cmd -ci -arch arm64 -noBuildRepoTasks -NoRestore -test -all
+                -projects eng\helix\helix.proj /p:IsHelixWindowsArmCheck=true
+                /p:IsHelixJob=true  /p:RunTemplateTests=true
+                /p:CrossgenOutput=false /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log
```

Comment:
Generally, be consistent w/ property settings in all three build steps. This line appears multiple times in the other job for example.